content/posts/gsoc-2022-x86-il/index.md

```diff
@@ -0,0 +1,125 @@
+---
+author: "DMaroo"
+title: "GSoC 2022 - x86 ISA lifting for RzIL"
+date: "2022-12-04"
+summary: "A summary of all the work done on RzIL lifting for x86 ISA, and how you can use it."
+tags: ["rizin", "gsoc"]
+ShowToc: false
+weight: 2
+---
+
+Hi! I'm [DMaroo](https://github.com/DMaroo), a GSoC 2022 mentee, working on IL migration for x86 ISA in Rizin. For the past few months, I have worked on implementing x86 instructions (from 8086, 80186, 80286 and 80386 instruction sets) in Rizin's intermediate language, [RzIL](https://github.com/rizinorg/rizin/blob/dev/doc/rzil.md).
+
+The following article covers all the work, design decisions, challenges and future plans of the work that I've been doing. The RzIL can be accessed using `aez` commands (RzIL emulation).
+
+## Relevant pull requests
+
+Implementation of RzIL lifting for x86 instructions: [rizinorg/rizin#2747](https://github.com/rizinorg/rizin/pull/2747)
```

Comment:
Since the PR was merged, maybe it's better to point to the commit and/or concrete files instead?